src/serviceregistry/5_0_0/sysd/serviceregistry_sysd.tex

```diff
@@ -0,0 +1,228 @@
+    \item \textbf{Service instance}: name, service definition and system reference, version, expiration date and time, instance level custom meta data, interface reference, security policy, and interface-related data (e. g. IP addresses, port, base path, content type, input and output data models).
+\end{itemize}
+
+\subsection{Important Delimitations}
+\label{sec:delimitations}
+
+\begin{itemize}
+    \item Names (device name, system name, service definition name, service instance name, interface name) must be unique in the Local Cloud.
+    \item If the Local Cloud does not contain an Authentication system or not using X.509 certificates, there is no way for the Service Registry to verify the requester system. In that case, the Service Registry will consider the authentication data comes from the requester as valid.
+    \item If Service Registry's lookup is strict, only "public" services (such as the \textit{orchestration} service) can be queried directly from the Service Registry by the application systems. For any "non-public" services the application systems need to orchestrate. "Public" services should be marked in the Service Registry, for example using a service instance meta data entry.
+\end{itemize}
+
+\newpage
+
+\section{Services produced}
+\label{sec:services}
+
+\msubsection{service}{service-discovery}
+The purpose of this service is to lookup, publish and revoke provided services. The service is offered for both application and core/support systems. 
+
+\msubsection{service}{system-discovery}
+The purpose of this service is to lookup, publish and revoke systems that are part of (or wants to be part of) the Local Cloud. The service is offered for both application and core/support systems. 
+
+\msubsection{service}{device-discovery}
+The purpose of this service is to lookup, publish and revoke devices on which the Local Cloud's systems are running. The service is offered for both application and core/support systems. 
+
+\msubsection{service}{service-registry-management}
+Recommended service.  Its purpose is to manage service definitions, service instances, interfaces, systems and devices in bulk. The different operations provide querying, registering and unregistering functionalities. The service is offered for administrative support systems. 
+
+\msubsection{service}{monitor}
+Recommended service. Its purpose is to give information about the provider system. The service is offered for both application and core/support systems.
+
+\newpage
+
+\section{Security}
+\label{sec:security}
+
+For authentication, the Service Registry can utilize an other core system, the Authentication system's service to verify the identities of the requester systems. If no Authentication system is deployed into the Local Cloud, the Service Registry can accept X.509 certificates for identification. If none of these are available, the Service Registry trusts the requester system self-provided identity.
```

Comment:
"the Service Registry trusts the requester system self-provided identity" -> "the Service Registry trusts the requester system'**s** self-provided identity"